src/main/java/frc/robot/Constants.java

```diff
@@ -259,5 +259,14 @@ public static class ShooterConstants {
     public static final PID kRightPID = new PID(0, 0, 0);
 
     public static final double kShootingTolerance = 0.05;
+    
+public static class ClimberConstants {
+  public static final int m_climberLeftMotorID = 20;
+  public static final int m_climberRightMotorID = 21;
+  //3000 rpm, 12 mm per rotation, needs to move 12 inches
+  public static final double kClimberVelocityConversionFactor = (0.2);
+  public static final PID kClimberPIDLeft =  new PID(0.5, 0.0, 0.0);
```

Comment:
You created the climber constants inside the shooter constants and there is still an indentation error